README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://modx.com/">
-    <img alt="MODX Revolution" src="https://modx.com/assets/design/css/images/modx-logo-color.svg" width="180" />
+    <img alt="MODX Revolution" src="https://modx.com/assets/themes/manana/images/logo.svg" width="180" />
```

Comment:
Would it be possible to have the logo available in the root like the favicon? 
This way we don't need to change the URL whenever the theme changes.

Reply:
It will be a mess. And if it is in the root, it may be deleted accidentally by users. I don't like solve such kind of issues. And another case – people may remove it but to blame the MODX.

Reply:
Hm, it is a path on the modx.com website. 
What if in this case just use a link to the file in the repo? Like this?
[](https://raw.githubusercontent.com/modxcms/revolution/2.x/manager/templates/default/images/modx-icon-color.svg) - https://raw.githubusercontent.com/modxcms/revolution/2.x/manager/templates/default/images/modx-icon-color.svg

Reply:
I went with the link to the file in the GitHub repository.